src/app/(admin)/admin/_hooks/useFindEntityMatchingPhoneOrEmail.js

```diff
@@ -0,0 +1,51 @@
+import { useGetList } from 'react-admin';
+import { RESOURCES } from '@admin/_lib/consts';
+import { z } from 'zod';
+import { PHONE_REGEX } from '@/lib/consts';
+
+export const useFindEntityMatchingPhoneOrEmail = ({ key, value }) => {
+  const validationSchema = {
+    phone: z
+      .string()
+      .trim()
+      .refine(val => PHONE_REGEX.test(val)),
+    email: z.string().trim().email(),
```

Comment:
phone and email keys should be get from consts you created, [PHONE]: z.string()...

Reply:
Fixed